CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# Docs:
+#   https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# ongoing discussions for below:
+#    - https://github.community/t/protecting-github-workflows/16122
+#    - https://github.com/actions/runner/issues/494
+
+.github/   @adobe/parliament-tc
```

Comment:
Can we make this `.github/workflows`? Folks may want to fix things in our issues or pr templates.

Reply:
sounds good!